config/locales/en.yml

```diff
@@ -1270,8 +1270,33 @@ en:
   footer_legal_tos: "Terms and conditions"
   footer_legal_visit: "Find us on"
   footer_legal_text_html: "Open Food Network is a free and open source software platform. Our content is licensed with %{content_license} and our code with %{code_license}."
-
+  footer_legal_data_text: "We take good care of your data." 
+  footer_legal_data_call:  "See our"
```

Comment:
I see why you split the text, but it makes it hard for translators to get the context. Depending on the grammar of the language, the link may also not be at the end of the sentence. I would do it like `footer_legal_text_html`:

```yml
footer_legal_data_statement_html: "We take good care of your data. See our %{cookies_policy}."
```

Reply:
It would also be nice to refactor these translations to use lazy lookup. I know, you went with the existing structure and that's fine. But it's even better when we can do a bit of refactoring on the way and improve the code that touching the new things we want to build.

Reply:
I did it this way because, on the same sentence, there is a conditional. We have to display privacy URL if it is configured in the BO.
See footer_legal_data_privacy_policy  here: https://github.com/openfoodfoundation/openfoodnetwork/pull/2411/commits/1e3b67d5da3ed2aaf7b80b3910aa5418c9961d24
With this in mind, I am not sure I can do what you suggest (in the requirements the privacy policy URL comes before the cookies policy in the sentence). Do you understand?

Reply:
In terms of lazy lookup, do you mean to move all footer translations (the ones from line 1223 on en.yml to line 1254) into the "shared" subfolder on en.yml (line 1085)? I think that's what you mean. Please confirm I'll do it.

Reply:
Maybe you could have both sentences in full as separate keys, and show either one based on the conditional instead of a split sentence.

Reply:
thanks for your feedback!
Agreed, I have made it 2 versions with the full sentence. Please have a look at the new version.